src/site/content/en/blog/receive-shared-data/index.md

```diff
@@ -0,0 +1,262 @@
+authors:
+  - petelepage
+  - joemedley
+date: 2019-11-08
+hero: hero.jpg
+alt: A hand sharing a web.dev article.
+description: |
+  On a mobile device, sharing should be as simple as clicking the Share button,
+  choosing an app, then choosing who to share with. The Web Share Target API
+  allows installed web apps to register with the underlying operating system to receive shared content.
+tags:
+  - post
+  - capabilities
+---
+
+On a mobile device, sharing should be as simple as clicking the **Share** button,
+choosing an app, and choosing who to share with. For example, you may want to
+share an interesting article, either by emailing it to friends or tweeting it to
+the world.
+
+In the past, only native apps could register with the operating system to
+receive shares from other installed apps. But with the Web Share Target API,
+installed web apps can register with the underlying operating system
+as a share target to receive shared content.
+Support for text and data was added in Chrome 71, and
+support for files was added in Chrome 76.
+
+{% Aside %}
+The Web Share Target API is only half of the magic. Web apps can share data,
+files, links, or text using the Web Share API. See
+[Share like a native](/share-like-a-native) for details.
+{% endAside %}
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="./wst-send.png" style="max-width: 400px;" alt="Android phone with the 'Share via' drawer open."/>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    System-level share target picker with an installed PWA as an option.
+  </figcaption>
+</figure>
+
+## See Web Share Target in action
+
+1. Using Chrome 76 or later, open the [Web Share Target demo][demo].
+2. When prompted, click **Install** to add the app to your home screen, or
+   use the Chrome menu to add it to your home screen.
+3. Open any app that includes a native share intent, or use the Share button
+   in the demo app.
+4. From the target picker, choose **Web Share Test**.
+
+After sharing to the demo app, you should see all of the information sent via
+the web share target web app.
+
+## Register your app as a share target
+
+To register your app as a share target, it needs to meet [Chrome's
+installability criteria][installability]. In addition, before a user can share
+to your app, they must add it to their home screen. This prevents sites from
+randomly adding themselves to the user's share intent chooser and ensures that
+sharing is something that users want to do with your app.
+
+## Update your web app manifest
+
+To register your app as a share target, add a `share_target` entry to its [web
+app manifest][manifest]. This tells the operating system to include your app as
+an option in the intent chooser. What you add to the manifest controls the data
+that your app will accept. There are three common scenarios for the `share_target`
+entry:
+- Accepting basic information
+- Accepting application changes
+- Accepting files
```

Comment:
The aside confused me for a second because it interrupts the flow of the section. I'm proposing this list as a way to signal structure to the reader. Feel free to cut it if you don't like it.